platform-emf/src/main/java/com/softicar/platform/emf/form/tab/factory/trait/EmfFormTraitTabFactory.java

```diff
@@ -24,8 +24,8 @@ public boolean isVisible(R tableRow) {
 		if (tableRow.impermanent()) {
 			return false;
 		} else {
-			T trait = traitTable.get(tableRow);
-			if (trait != null) {
+			T trait = traitTable.load(tableRow);
+			if (trait != null && !trait.impermanent()) {
```

Comment:
- i don't this `load` will ever return an impermanent row (we should check that, adapt Javadoc and add a test)
- a test is missing for this change